src/domains/idpay-app/12_alert.tf

```diff
@@ -264,3 +264,101 @@ data
 
   tags = var.tags
 }
+
+resource "azurerm_monitor_scheduled_query_rules_alert_v2" "GeneratePaymentFile_csv_zip" {
+
+  count               = var.idpay_alert_enabled ? 1 : 0
+  name                = "${var.prefix}-GeneratePaymentFile_csv_zip"
+  location            = data.azurerm_resource_group.monitor_rg.location
+  resource_group_name = data.azurerm_resource_group.monitor_rg.name
+
+  evaluation_frequency = "P1D"
+  window_duration      = "P1D"
+  scopes               = [data.azurerm_log_analytics_workspace.log_analytics.id]
+  severity             = 0
+  criteria {
+    query                   = <<-QUERY
+  let startTime = ago(365d);
```

Comment:
@LarissaASLeite this `startTime` is suspect: it goes too back in time. As far as I can remember, you can't exceed 48h. Please check this reference https://github.com/MicrosoftDocs/azure-docs/issues/22637#issuecomment-455050406

Reply:
@GiovanniMancini I fixed it changing to 1d